src/Textarea.jsx

```diff
@@ -261,7 +259,7 @@ class ReactTextareaAutocomplete extends React.Component<
 
     providedData
       .then(data => {
-        if (typeof providedData !== 'object') {
```

Comment:
I think `data` is what should be checked to be an array, `providedData` will always be a object if it's a Promise

Reply:
Yeah! You are definitely right. I'm just wondering why not use `Array.isArray`?

Reply:
Yeah! You are definitely right. I'm just wondering why not to use `Array.isArray`?

Reply:
`Array.isArray` isn't supported on IE8, but I just checked and React 15 and up don't support IE8 anyways so I'll update to `Array.isArray`.

Reply:
Yes. Thanks! Anyway, we do transpile all the code, so Babel does it for us in every case.